howto/application/virtual-devices.md

```diff
@@ -79,7 +71,46 @@ instance-type: a4.3
 addons: [vdev-support]
 ```
 
-The application will now include the [Lawnchair](https://lawnchair.app/) and has it configured as the default system launcher.
+### Extend the Application with Hooks
```

Comment:
I feel we mix too many things here and should either use an addon or put everything into the app package. The howto is meant to focus on a virtual device so lets concentrate on that and drop the vdev-support addon part entirely.

Reply:
Okay.  I'm fine with that.
@ru-fu how do you think?

Reply:
hmm, I am a bit reluctant to do so, which creates a new page to introduce two ways to extend an application. 
1. The content under the `## Extend an applicaiton with addons` may be duplicated with some parts of https://anbox-cloud.io/docs/tut/creating-addon . We can create a link here and refer to creating-addon page. But that's not our initial intention here.
2. We need an example to explain how to extend an application with hooks, [howto/application/virtual-devices.md](https://github.com/anbox-cloud/discourse-docs/pull/127/files#diff-18dabe8c190d7f390fb2d98717ecfee458d9146eaf6011f2a577cf35fe97da32) is the best fit for this so people could understand the difference between both appoarchs for application extension.

After thinking twice, I feel like we may leave it as is.
Anyway, let me know your opinion about this.
Thanks.

Reply:
I agree that we need to explain how to extend an application with hooks, but I don't feel "How to create a virtual Android device" is the right place for it - that how-to is about an "empty" application, and it's probably more likely that hooks are used with "real" applications. And as a user, I wouldn't look for information on how to extend my application in the page about virtual devices.

So I think we should have a new page for this. I don't like the duplication of the addon content either, so for now we could probably just explain that there are two ways of doing this, either with an addon as described in [link to tutorial] or with hooks.
(Eventually, we should probably move the addon tutorial to the how to.)

Reply:
Okay, that makes sense.